tests/0054-offset_time.cpp

```diff
@@ -130,6 +130,22 @@ static void test_offset_time (void) {
     fails += verify_offset(query_parts[1], timestamps[ti], timestamps[ti+1], RdKafka::ERR_NO_ERROR);
   }
 
+   /* repeat test with -1 timeout */
+  for (int ti = 0 ; ti < timestamp_cnt*2 ; ti += 2) {
+    RdKafka::TopicPartition::destroy(query_parts);
+    query_parts.push_back(RdKafka::TopicPartition::create(topic, 0, timestamps[ti]));
+    query_parts.push_back(RdKafka::TopicPartition::create(topic, 1, timestamps[ti]));
+
+    Test::Say(tostr() << "Attempting offsetsForTimes() for timestamp " << timestamps[ti] << "\n");
+    err = p->offsetsForTimes(query_parts, tmout_multip(-1));
```

Comment:
the timeout multiplier should not be used here since we want a specific timeout.